planning/behavior_velocity_planner/autoware_behavior_velocity_blind_spot_module/include/autoware/behavior_velocity_blind_spot_module/time_to_collision.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2025 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2025 TIER IV, Inc.
```

Reply:
fixed in f3d3f9802c8a7424d5775c7e7155f65633b8dc83, let me close